### ISSUE TITLE
Support --no-color

### ISSUE THREAD

**Issue by pop** [helix-editor/helix#1923]
#### Describe your feature request

Helix is great! When I use it though I'd like to be able to disable colors.
For example, when running `hx --health` I'd like to be able to pipe it to `less` without seeing color-codes.

Th option could be passed as a flag `--no-color` to some or all helix commands.
It can also be parsed from the environment via the `NO_COLOR` environment variable.

See https://no-color.org/ for more info.

I'm happy to work on this issue if somebody can point me where to start.

**Comment by NNBnh** [comment]
Good idea! Good idea! #1074 might be related.

**Comment by pascalkuthe** [comment]
I don't think we will add support for this. Helix fundamentally needs escape sequences to work (just for render diffing alone and that will never change). We also need to set the selection (and cursors since there are multiple) to a white color too so it's visible. 

So an option that fully disables all escape sequences and colors doens't make sense. The closest you can realistically get in helix is a black/white theme. That is just another theme (that you can already easily create yourself tough) and enabling it in your config (or with `:theme <name >`) seems reasonable and doens't need special support.

Crossterm also has hardcoded support for `NO_COLOR` (that we can't turn off) which completely disables emitting any colors codes. This pretty much makes helix unusable (see #8071). Since we can't turn that off upstream we can't support the environment variable even if we wanted to.

Ultimately I think NO_COLOR just doesn't make much sense for an interactive tui like Helix.
